Corona-Warn-App/src/main/java/de/rki/coronawarnapp/service/applicationconfiguration/ApplicationConfigurationService.kt

```diff
@@ -14,8 +14,8 @@ object ApplicationConfigurationService {
                     .clearSupportedCountries()
                     .addAllSupportedCountries(
                         listOf(
-                            "DE", "UK", "FR", "IT", "SP", "PL", "RO", "NL",
-                            "BE", "CZ", "EL", "SE", "PT", "HU", "AT", "CH", "BG", "DK", "FI", "SK",
+                            "DE", "UK", "FR", "IT", "ES", "PL", "RO", "NL",
+                            "BE", "CZ", "SE", "PT", "HU", "AT", "CH", "BG", "DK", "FI", "SK",
```

Comment:
How about using constants (of Locale?) for this one instead of hardcoding strings?

Reply:
Those are just for Mock purpose. Our backend is currently only providing the list of country codes in dev. If it is implemented in production we just have to remove this list.